### DISCUSSION TITLE
Redevelopment of HAGIWO clones

### DISCUSSION THREAD

**Discussion by mzuelch** [mzuelch/CATs-Eurosynth#10] [Ideas]
In my HAGIWO clones, I tried to stay as close as possible to the original - apart from converting it to typical/standardised voltages and some minor tweaks. One of the main reasons was the catastrophic availability of bare microcontrollers during the last few years.
The use of a complete Nano led to some restrictions in PCB design and prevented me from adding some nifty features beyond the original design.

It looks like the microcontroller shortage has come to an end (a least for ATMEGA328). Starting with Sync LFO, I plan to finally redesign the schematic and PCB to bare microcontroller and publish it on this GitHub as a separate design.

I'm about to collect ideas for further features and gimmicks for the SyncLFO module and would be happy for some contributions from the community.
Please leave a post with your ideas.
And yes - bicolour LED, changes to the front design etc. are already on my ToDo list.

**Comment by modulove** [comment]
Quite like idea of a bare microcontroller design to free some space.

I was thinking to add two cv inputs as well as a button for swithing modes / waveform / firmware maybe.

For the Firmware i think a low pass filter on the self-modulation or some tuning would be good to tame it some little bit.
